Gemfile

```diff
@@ -67,4 +67,9 @@ group :lint do
   gem 'rubocop', require: false
   gem 'rubocop-performance', require: false
   gem 'rubocop-rails', require: false
+
+  gem 'prettier_print', require: false
```

Comment:
what are these gems for? Does Limber still use haml for views? I always tend to be cautious about adding more gems as they can add to technical debt. It would at least be useful to add comments as to why they are here so if they start causing an issue we know why they are here.